src/less/resources.less

```diff
@@ -273,6 +273,16 @@
   &.has-avatar {
     align-items: center;
   }
+
+  .rhinobox__input {
+    position: absolute !important;
+    -webkit-appearance: none;
+    height: 2rem !important;
+    width: 2rem !important;
+    z-index: 99999;
```

Comment:
It looks like this project is using `z-index: 1000` to indicate the highest z-index value.